docs/edgeql/functions/converters.rst

```diff
@@ -49,3 +385,251 @@ the functions below allow more options for such conversions.
+| Q                       | quarter                                |
++-------------------------+----------------------------------------+
+| RM                      | month in upper case Roman numerals     |
+|                         | (I-XII; I=January)                     |
++-------------------------+----------------------------------------+
+| rm                      | month in lower case Roman numerals     |
+|                         | (i-xii; i=January)                     |
++-------------------------+----------------------------------------+
+| TZ                      | upper case time-zone abbreviation      |
+|                         | (only supported in to_char)            |
++-------------------------+----------------------------------------+
+| tz                      | lower case time-zone abbreviation      |
+|                         | (only supported in to_char)            |
++-------------------------+----------------------------------------+
+| TZH                     | time-zone hours                        |
++-------------------------+----------------------------------------+
+| TZM                     | time-zone minutes                      |
++-------------------------+----------------------------------------+
+| OF                      | time-zone offset from UTC (only        |
+|                         | supported in to_char)                  |
++-------------------------+----------------------------------------+
+
+Some additional formatting modifiers:
+
++---------------+-----------------------------------+---------------+
+| Modifier      | Description                       | Example       |
++===============+===================================+===============+
+| FM prefix     | fill mode (suppress leading       | FMMonth       |
+|               | zeroes and padding blanks)        |               |
++---------------+-----------------------------------+---------------+
+| TH suffix     | upper case ordinal number suffix  | DDTH, e.g.,   |
+|               |                                   | 12TH          |
++---------------+-----------------------------------+---------------+
+| th suffix     | lower case ordinal number suffix  | DDth, e.g.,   |
+|               |                                   | 12th          |
++---------------+-----------------------------------+---------------+
+| FX prefix     | fixed format global option (see   | FX Month DD   |
+|               | usage notes)                      | Day           |
++---------------+-----------------------------------+---------------+
+
+Normally when parsing a string input whitespace is ignored, unless
+the *FX* prefix modifier is used. For example:
+
+.. code-block:: edgeql-repl
+
+    db> SELECT to_naive_date(
+    ...     '2000    JUN', 'YYYY MON');
+    {<naive_date>'2000-06-01'}
+    db> SELECT to_naive_date(
+    ...     '2000    JUN', 'FXYYYY MON');
+    InternalServerError: invalid value "   " for "MON"
```

Comment:
Ideally we should fix `errmech.py` to recognize Postgres' error and convert it to a proper EdgeDB error.  Please add a new issue for that.